website/pages/docs/post-processors/docker-import.mdx

```diff
@@ -90,6 +155,30 @@ from ubuntu as an simple example:
 }
 ```
 
+</Tab>
+<Tab heading="HCL2">
+
+```hcl
+post-processor "docker-import" {
+  repository = "local/centos6"
+  tag = "latest"
+  changes = [
+    "USER www-data",
+    "WORKDIR /var/www",
+    "ENV HOSTNAME www.example.com",
+    "VOLUME /test1 /test2",
+    "EXPOSE 80 443",
+    "LABEL version=1.0",
+    "ONBUILD RUN date",
+    "CMD [\"nginx\", \"-g\", \"daemon off;\"]",
+    "ENTRYPOINT /var/www/start.sh"
```

Comment:
```suggestion
    "ENTRYPOINT /var/www/start.sh",
```
This also works ! But then adding the comma allows to now have a git diff when adding a line after.